pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -368,59 +357,8 @@ func (r *ReconcileGitopsService) reconcileDefaultArgoCDInstance(instance *pipeli
 		} else {
 			return reconcile.Result{}, err
 		}
-	} else {
-		changed := false
-
-		if existingArgoCD.Spec.ApplicationSet != nil {
```

Comment:
Why are we deleting all these?  We are still setting resource limits on the workloads, correct?

Reply:
for upgrades to v1.2 , we forcefully added this logic to ensure all the resources have requests/limits as the default namespace has resourcequota. Since, we removed the resourcequota object from v1.2.1 on default namespace this forceful updation is not needed.

Moreever, we dont know how many applications is the instance managing with the previous versions. So, these defaults might give a breaking experience.

Reply:
I thought we were removing quota in the namespace and leaving resource requests/limits (with correct defaults) in the workloads.   When the user want to enforce quota, he/she can do so by adding quota in the ArgoCD CR.

Reply:
For the fresh install yes I followed the same approach but for the upgrades I choose not to force any resource limits as we don't know the size of their applications. We can discuss on this for consensus

Reply:
For upgrade existing argocd workloads with resource limits/resources, this PR would have no effect on them as we only increase the default memory limit for repo server.   The default will not be to  any existing workload that already has resource limit configured.

Reply:
Ack. Reverted the changes and tests.

Reply:
@wtam2018 PR is ready for review again.